oshi-core/src/main/java/oshi/hardware/platform/unix/freebsd/FreeBsdCentralProcessor.java

```diff
@@ -207,15 +207,16 @@ private static List<LogicalProcessor> parseTopology() {
 
                     // Regex guarantees parsing digits so we won't get a
                     // NumberFormatException
+                    long parsedVal = ParseUtil.hexStringToLong(firstVal, 0);
```

Comment:
Interesting.... the comments indicate the regex guarantees the parsing won't fail.  But at the same time it's more readable with the hex string parsing. But then when you look at that method it's a lot of manual manipulation when I think the actual parsing-with-radix would handle it!

Anyway, probably ok here.